deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml

```diff
@@ -43,8 +45,8 @@ spec:
           image: quay.io/k8scsi/csi-snapshotter:v1.1.0
           args:
             - "--csi-address=$(ADDRESS)"
-            - "--connection-timeout=15s"
             - "--v=5"
+            - "--timeout=60s"
```

Comment:
note: snapshot default timeout  is 1 minute

https://github.com/kubernetes-csi/external-snapshotter/blob/master/cmd/csi-snapshotter/main.go#L54

Reply:
Yes, please see above comment on default disagreeing with the text for the option: "The timeout for any RPCs to the CSI driver. Default is 10s." and hence ensuring that this stays even if the snapshotter gets fixed one way or the other.

Reply:
Resolved, as per discussion on the timeout comment.